jso/apis/src/main/java/org/teavm/jso/util/function/JSConsumer.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright 2023 Bernd Busse.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.jso.util.function;
+
+import java.util.Objects;
+
+@FunctionalInterface
+public interface JSConsumer<T> {
+    void accept(T t);
+
+    default JSConsumer<T> andThen(JSConsumer<? super T> after) {
```

Comment:
Not sure JSO will handle this properly. Have you checked it?

Reply:
That's what my question about automated tests was about. :wink:
I'm in the process of writing some manual tests for the added functionality.